manifests/k/ktsu/BuildMonitor/1.0.0.0/ktsu.BuildMonitor.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ktsu.BuildMonitor
+PackageVersion: 1.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/ktsu-dev/BuildMonitor/releases/download/1.0.0-alpha.1-win-x64/BuildMonitor-1.0.0-alpha.1-win-x64.exe
+  InstallerSha256: 229EB9C552DFFCC2E395695991484C2E6ECC1CA660A02D940636934EDDCDEC82

```

```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/ktsu-dev/BuildMonitor/releases/download/1.0.0-alpha.1-win-x64/BuildMonitor-1.0.0-alpha.1-win-x64.exe
  InstallerSha256: 229EB9C552DFFCC2E395695991484C2E6ECC1CA660A02D940636934EDDCDEC82
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

_Microsoft Visual C++ Redistributable 2015-2022 (x64)_ is necessary here.

---

I will put a step to install the redist in my installer, thankyou!